scripts/linters/pylint_extensions_test.py

```diff
@@ -3340,3 +3340,183 @@ def __init__(self, fake_arg):
 
         with checker_test_object.assertNoMessages():
             checker_test_object.checker.visit_classdef(metaclass_node)
+
+
+class DisallowHandlerWithoutSchemaTests(unittest.TestCase):
+
+    def setUp(self):
+        super(DisallowHandlerWithoutSchemaTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.DisallowHandlerWithoutSchema)
+        self.checker_test_object.setup_method()
+
+    def test_schema_handlers_without_request_args_raise_error(self):
+
+        schemaless_class_node = astroid.extract_node(
+            """
+            class BaseHandler():
+                HANDLER_ARGS_SCHEMAS = None
+                URL_PATH_ARGS_SCHEMAS = None
+
+            class FakeClass(BaseHandler):
+                URL_PATH_ARGS_SCHEMAS = {}
+            """)
+        with self.checker_test_object.assertAddsMessages(
+            testutils.Message(
+                msg_id='no-schema-for-handler-args',
+                node=schemaless_class_node,
+                args=(schemaless_class_node.name)
+            )
+        ):
+            self.checker_test_object.checker.visit_classdef(
+                schemaless_class_node)
+
+    def test_schema_handlers_without_url_path_args_raise_error(self):
+
+        schemaless_class_node = astroid.extract_node(
+            """
+            class BaseHandler():
+                HANDLER_ARGS_SCHEMAS = None
+                URL_PATH_ARGS_SCHEMAS = None
+
+            class FakeClass(BaseHandler):
+                HANDLER_ARGS_SCHEMAS = {}
+            """)
+
+        with self.checker_test_object.assertAddsMessages(
+            testutils.Message(
+                msg_id='no-schema-for-url-path-elements',
+                node=schemaless_class_node,
+                args=(schemaless_class_node.name)
+            )
+        ):
+            self.checker_test_object.checker.visit_classdef(
+                schemaless_class_node)
+
+    def test_handlers_with_schema_do_not_raise_error(self):
+
+        schemaless_class_node = astroid.extract_node(
+            """
+            class BaseHandler():
+                HANDLER_ARGS_SCHEMAS = None
+                URL_PATH_ARGS_SCHEMAS = None
+
+            class FakeClass(BaseHandler):
+                URL_PATH_ARGS_SCHEMAS = {}
+                HANDLER_ARGS_SCHEMAS = {'GET': {}}
+            """)
+
+        with self.checker_test_object.assertNoMessages():
+            self.checker_test_object.checker.visit_classdef(
+                schemaless_class_node)
+
+    def test_list_of_non_schema_handlers_do_not_raise_errors(self):
```

Comment:
[Optional]
I think here we should swap the actual list with a list containing our fake class name to avoid dependency on actual list in future if some changes were to occur. The current test also seems fine, but I just think it would a better practice in writing tests. @DubeySandeep wdyt?